exopy_pulses/tasks/tasks/instrs/transfer_pulse_loop_task.py

```diff
@@ -0,0 +1,299 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by ExopyHqcLegacy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Task to transfer a sequence on an AWG.
+
+"""
+import os
+from traceback import format_exc
+from pprint import pformat
+from collections import OrderedDict
+
+import numpy as np
+from atom.api import (Value, Str, Float, Typed, Bool, set_default, Enum)
+
+from exopy.tasks.api import (InstrumentTask)
+from exopy.utils.atom_util import ordered_dict_from_pref, ordered_dict_to_pref
+
+
+class TransferPulseLoopTask(InstrumentTask):
+    """Build and transfer a pulse sequence to an instrument.
+
```

Comment:
Since this make several several hypothesis on the driver it could be nice to list them.